### DISCUSSION TITLE
What's the expected result from examples/Burgers.py?

### DISCUSSION THREAD

**Discussion by piyueh** [lululxvi/deepxde#668] [Q&A] [answered]
Hi, I'm trying the code in examples folder to make sure my environment setup is correct (e.g., dependencies, versions, etc.). I tried `Burgers.py`. It ran without error messages (except some deprecation warnings from TensorFlow). But how to know whether the result is what I should expect?

Or if there's something like unit tests for users to know whether the installation works correctly?

This is the figure I got from `Burgers.py`:
![figure_2](https://drive.google.com/uc?export=view&id=13sAOFYJbdvX5qtF2oseP2p_Mxwfj_yCK)

Thanks!!

**Comment by lululxvi** [comment, marked as answer]
Your result is correct. If the code can successfully run without error, then the setup should be correct.

For Burgers, the input variable is two dimensional `(x, t)`, and the output is `y`, so the function surface is 3D. But, by default, the code projects the 3D plot into a 2D figure by ignore `t`. That is why the plot looks messy. The result is saved to `test.dat` with the first column `x`, the second column `t`, the third column `y_true`, and the fourth column `y_pred`. You can plot the result in 3D to compare the exact solution and prediction.

You can also try simples cases, such as `Poisson` in 1D.
